pkg/flowkit/gateway/emulator.go

```diff
@@ -38,10 +39,11 @@ import (
 )
 
 type EmulatorGateway struct {
-	emulator *emulator.Blockchain
-	backend  *backend.Backend
-	ctx      context.Context
-	logger   *logrus.Logger
+	emulator      *emulator.Blockchain
+	backend       *backend.Backend
+	ctx           context.Context
+	logger        *logrus.Logger
+	emultorLogger *zerolog.Logger
```

Comment:
These loggers should be combined, but this is probably work for another PR.

Reply:
Agreed

Reply:
Also agreed 😄